cmd/kueuectl/app/cmd.go

```diff
@@ -52,7 +52,7 @@ func NewDefaultKueuectlCmd() *cobra.Command {
 
 func NewKueuectlCmd(o KueuectlOptions) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "kueue",
+		Use:   "kueuectl",
```

Comment:
why change this?

Will it work for autocompletion when you don't have an alias (`kubectl kueue`)?

Reply:
No, it's not working with `kubectl kueue`. 
Also it's not working with `kueuectl` without this changes

Reply:
How to make it work for `kubectl kueue`? That will be the first way users will use this binary.

Then we can check how we can make it support both.

Reply:
For `kubectl kueue` we need to add `kubectl_complete-<pluginName>` file to $PATH directory. I will add it to docs.